appveyor.yml

```diff
@@ -18,6 +18,11 @@ init:
       $env:APPVEYOR_CACHE_SKIP_SAVE = "true"
       Exit-AppVeyorBuild
     }
+    # Non-PRs: skip MSVC_32 run for PRs already.
+    if ($env:APPVEYOR_PULL_REQUEST_HEAD_COMMIT -and $env:CONFIGURATION -match '^(MSVC_32)$') {
+      $env:APPVEYOR_CACHE_SKIP_SAVE = "true"
+      Exit-AppVeyorBuild
```

Comment:
`if ($env:APPVEYOR_PULL_REQUEST_HEAD_COMMIT)` would only be true for PRs, I think? I would think this should check the negation.